installer/pom.xml

```diff
@@ -248,4 +246,4 @@
             </plugin>
         </plugins>
     </reporting>
-</project>
+</project>
```

Comment:
Can we add the newline back to the end of this file?

Reply:
Done